worker/pkg/benthos/transformers/generate_ip_address_test.go

```diff
@@ -195,9 +187,9 @@ func Test_IpV4Public_NoOptions(t *testing.T) {
 	assert.False(t, isReservedIP(parsedIP), "should not be reserved IP")
 }
 
-func Test_IpV4PrivateA_Class(t *testing.T) {
-	class := "GENERATE_IP_ADDRESS_CLASS_PRIVATE_A"
-	mapping := fmt.Sprintf(`root = generate_ip(class:%q)`, class)
+func Test_IpV4PrivateA(t *testing.T) {
+	ipType := string(IpV4_PrivateA)
+	mapping := fmt.Sprintf(`root = generate_ip(ip_type:%q)`, ipType)
```

Comment:
Can we also have a bloblang test here that provides no inputs to ensure that it always works with defaults?

`mappings := "root = generate_ip()"`